fragments/filter/README.md

```diff
@@ -0,0 +1,786 @@
+      - [Example 1: POST with cql-json](#example-1-post-with-cql-json)
+    - [Example 2](#example-2)
+      - [Example 2: GET with cql-text](#example-2-get-with-cql-text)
+      - [Example 2: POST with cql-json](#example-2-post-with-cql-json)
+    - [Example 3](#example-3)
+      - [Example 3: GET with cql-text](#example-3-get-with-cql-text)
+      - [Example 3: POST with cql-json](#example-3-post-with-cql-json)
+    - [Example 4](#example-4)
+      - [Example 4: AND cql-text (GET)](#example-4-and-cql-text-get)
+      - [Example 4: AND cql-json (POST)](#example-4-and-cql-json-post)
+    - [Example 5](#example-5)
+      - [Example 5: OR cql-text (GET)](#example-5-or-cql-text-get)
+      - [Example 5: OR cql-json (POST)](#example-5-or-cql-json-post)
+    - [Example 6: Temporal](#example-6-temporal)
+      - [Example 6: ANYINTERACTS cql-text (GET)](#example-6-anyinteracts-cql-text-get)
+      - [Example 6: ANYINTERACTS cql-json (POST)](#example-6-anyinteracts-cql-json-post)
+    - [Example 6: Spatial](#example-6-spatial)
+      - [Example 6: INTERSECTS cql-text (GET)](#example-6-intersects-cql-text-get)
+      - [Example 6: INTERSECTS cql-json (POST)](#example-6-intersects-cql-json-post)
+
+## Overview
+
+The Filter extension provides an expressive mechanism for searching based on Item attributes.
+
+This extension references behavior defined in the 
+[OGC API - Features - Part 3: Filtering and the Common Query Language (CQL)](https://portal.ogc.org/files/96288)
+specification. As of May 2020, this specification is in draft status. The only anticipated change before final is to the 
+division of behavior among conformance classes, as described further 
+in the [Conformance Classes](#conformance-classes) section. 
+
+OAFeat Part 3 CQL formally defines syntax for both a text format (cql-text) as an ABNF grammar (largely similar to the BNF grammar 
+in the General Model) and a JSON format (cql-json) as a JSON Schema and OpenAPI schema, and provides a precise natural 
+language description of the declarative semantics.  The CQL Text format has long-standing use within 
+geospatial software (e.g., GeoServer), is expected not to change before final. 
+OGC CQL Text has been previously described in [OGC Filter Encoding](https://www.ogc.org/standards/filter) and 
+[OGC Catalogue Services 3.0 - General Model](http://docs.opengeospatial.org/is/12-168r6/12-168r6.html#62) 
+(including a BNF grammar in Annex B). The CQL JSON format is newly-defined, but also not
+expected to change before final.
+
+It should be noted that the "CQL" referred to here is OGC CQL. It is **not** referencing or related two other "CQL" languages, 
+the [SRU (Search/Retrieve via URL) Contextual Query Language](https://www.loc.gov/standards/sru/cql/index.html) (formerly 
+known as Common Query Language) or the [Cassandra Query Language](https://cassandra.apache.org/doc/latest/cql/) used by the 
+Cassandra database.
+
+## Limitations of Item Search 
```

Comment:
This section is more a justification for the extension, and perhaps doesn't need to be so prominent. It was a little confusing to me reading through the spec as I was expecting this section to begin talking about what this extensions supports. I'd recommend making a section lower down dedicated to talking about other parts of the spec in comparison or justifying this extension, and have the beginning of this document focused on specifying what the filter extension is and does.

Reply:
I'll think about that. I like having the "motivation" section first, to adequately set the stage for where we're starting with this extension.